### DISCUSSION TITLE
Feedback/suggestions before the final relase of OpenAttestation v4.0

### DISCUSSION THREAD

**Discussion by HJunyuan** [Open-Attestation/open-attestation#309] [General]
## Context
Following earlier discussions in #249 to make the OpenAttestation framework compliant with [W3C Verifiable Credentials Data Model v2.0](https://www.w3.org/TR/vc-data-model-2.0/), a new version of the framework has been in the works: [OpenAttestation v4.0 (beta)](https://github.com/Open-Attestation/open-attestation/tree/beta)

## At a glance
- Documents issued with OA v4.0 now aligns with W3C VCDM v2.0 ([DID-issued sample](https://github.com/Open-Attestation/open-attestation/blob/beta/test/fixtures/v4/__generated__/signed-wrapped-document-did.json))
- `renderMethod` now aligns with the upcoming draft of [W3C VC Rendering Methods](https://w3c-ccg.github.io/vc-render-method/#openattestationembeddedrenderer)
- For more details, refer to Action 1a, 1b, 1e in #249

## Feedback or suggestions
This thread serves as a place for anyone to submit feedback or suggestions before the final release of OpenAttestation v4.0. Please start a new comment thread for each topic.

**Comment by cxcheng** [comment]
The OpenAttestation team has been discussing internally that the proof type "OAMerkleProofSignature2018" under OA v4 data model does not correctly reflect the fact that we are only applying hash of hashes rather than Merkle Tree for our algorithm. As part of our effort to work with W3C and promote our proof method, we feel it is important that we don't mislead regarding Merkle Tree. I am hereby proposing the type "OpenAttestationHashProof2018". Do you have objections on that, or have an alternate name to propose?
